.github/workflows/acceptance-tests-runner.yml

```diff
@@ -217,27 +192,52 @@ jobs:
             - 'internal/service/orginvitation/*.go'
             - 'internal/service/organization/*.go'
             - 'internal/service/thirdpartyintegration/*.go'
+          data_lake:
+            - 'internal/service/datalakepipeline/*.go'
+          encryption:
+            - 'internal/service/encryptionatrest/*.go'  
           event_trigger:
-            - 'internal/service/eventtrigger/*.go'
-          search_index:
-            - 'internal/service/searchindex/*.go'
+            - 'internal/service/eventtrigger/*.go'  
           federated:
             - 'internal/service/federatedsettingsidentityprovider/*.go'
             - 'internal/service/federateddatabaseinstance/*.go'
             - 'internal/service/federatedsettingsorgconfig/*.go'
             - 'internal/service/federatedsettingsorgrolemapping/*.go'
-            - 'internal/service/federatedquerylimit/*.go'
-          data_lake:
-            - 'internal/service/datalakepipeline/*.go'
+            - 'internal/service/federatedquerylimit/*.go'  
+          generic:
+            - 'internal/service/backupcompliancepolicy/*.go'
+            - 'internal/service/auditing/*.go'
+            - 'internal/service/x509authenticationdatabaseuser/*.go'  
           ldap:
             - 'internal/service/ldapconfiguration/*.go'
-            - 'internal/service/ldapverify/*.go'
-          encryption:
-            - 'internal/service/encryptionatrest/*.go'
-        
-  cluster_outage_simulation:
+            - 'internal/service/ldapverify/*.go'  
+          network:
+            - 'internal/service/networkcontainer/*.go'
+            - 'internal/service/networkpeering/*.go'
+            - 'internal/service/privateendpointregionalmode/*.go'
+            - 'internal/service/privatelinkendpoint/*.go'
+            - 'internal/service/privatelinkendpointservice/*.go'
+            - 'internal/service/privatelinkendpointservicedatafederationonlinearchive/*.go' 
+          project:
+            - 'internal/service/project/*.go'
```

Comment:
[nit] Could we sort these too? 😅

Reply:
it's sorted, isn't it?

Reply:
i used the same order for the 3 parts: filter outputs, filter files and jobs

Reply:
e.g.,:
```yaml
          project:
            - 'internal/service/accesslistapikey/*.go'
            - 'internal/service/project/*.go'
            - 'internal/service/projectinvitation/*.go'  
            - 'internal/service/projectipaccesslist/*.go'
```

Reply:
you mean files and packges inside each test group?  good catch!

Reply:
Currently, 
![image](https://github.com/mongodb/terraform-provider-mongodbatlas/assets/6596097/3f5eb15f-3426-4dab-80c7-84a1349f307f)